pynecone/utils.py

```diff
@@ -799,17 +799,31 @@ def run_backend_prod(
     setup_backend()
 
     num_workers = get_num_workers()
-    command = constants.RUN_BACKEND_PROD + [
-        "--bind",
-        f"0.0.0.0:{port}",
-        "--workers",
-        str(num_workers),
-        "--threads",
-        str(num_workers),
-        "--log-level",
-        str(loglevel),
-        f"{app_name}:{constants.APP_VAR}()",
-    ]
+    command = []
```

Comment:
Don't need to define the variable here, we can just do it within the if/else statements